tests/optimizers/gulp/test_uff_assign_ff.py

```diff
@@ -210,29 +174,6 @@ def __init__(self, molecule, atom_types, has_metal, metal_ff):
             has_metal=False,
             metal_ff=None,
         ),
-        # CaseData(
```

Comment:
Please reinstate these in some way. These are tests that I know will fail based on a known issue.